### PR TITLE
[TITAN-145] - Prepare Documentation for Connector Plugin

### PR DESCRIPTION
**Pull request by RossoMaguire**
### Description 

Adding documentation for the Atlas Commerce Connector for BigCommerce plugin that will come pre installed as part of the eCommerce Blueprint. 

[Preview app to review the docs on the site ](https://h2wt5zfin2hzahss2jnqonnka.js.wpenginepowered.com/docs/atlas-commerce-connector)

[TITAN-145](https://wpengine.atlassian.net/jira/software/c/projects/TITAN/boards/1122?modal=detail&selectedIssue=TITAN-145)

[TITAN-145]: https://wpengine.atlassian.net/browse/TITAN-145?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: atlas-commerce-connector/introduction.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+slug: /atlas-commerce-connector/introduction
+title: Atlas Commerce Connector for BigCommerce - Introduction
+description: Connect BigCommerce product data to WordPress.
```
Comment (alvarocavalcanti) [inline atlas-commerce-connector/introduction.mdx:4]: Are these supposed to be rendered? I understood those would work like metadata, but the GitHub preview displays them as a table.

File: atlas-commerce-connector/tutorial/connecting-bigcommerce.mdx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+slug: /atlas-commerce-connector/tutorial/connecting-bigcommerce
+title: Atlas Commerce Connector for BigCommerce - Connecting BigCommerce
+description: Learn how to connect BigCommerce API accounts to WordPress.
+---
+
+**Note**: These tutorials assume that you have already installed the [Atlas Commerce Blueprint via our User Portal](#)
```
Comment (alvarocavalcanti) [inline atlas-commerce-connector/tutorial/connecting-bigcommerce.mdx:7]: Maybe link to the first document instead?
```suggestion
**Note**: These tutorials assume that you have already installed the [Atlas Commerce Blueprint via our User Portal](introduction.mdx)
```

File: atlas-commerce-connector/introduction.mdx (hunk 1 above)
Comment (RossoMaguire) [reply atlas-commerce-connector/introduction.mdx:4]: Yeah they will be rendered by developers.wpengine.com see Atlas Content Modeler for an example which is the dir I was going off: https://developers.wpengine.com/docs/atlas-content-modeler/introduction 

There are also contribution guides here: 
https://github.com/wpengine/headless-docs/blob/develop/CONTRIBUTING.md

File: atlas-commerce-connector/tutorial/connecting-bigcommerce.mdx (hunk 2 above)
Comment (RossoMaguire) [reply atlas-commerce-connector/tutorial/connecting-bigcommerce.mdx]: That makes more sense! On the introduction page we can fill in the link to the marketing / landing / sign up material for the Blueprint / Sandbox accounts. I'll ask Bryan for that at stand up

**Review by JEverhart383 — CHANGES_REQUESTED** [review]
@RossoMaguire thanks for getting these together so we can work through things before GA. Since this adds an additional doc section, there are a few additional things we'll need to do. 

1. Can you add a config.json file for this new docs section so that the site can ingest them? You can look at this file in the ACM folder as an example: https://github.com/wpengine/headless-docs/blob/feature/atlas-commerce-docs/atlas-content-modeler/config.json
2. I'll need to create a PR into wpengine/developers to add this docs section to the site to make it accessible. Once there is a config file, the direct URLs should be accessible.
3. I created a preview environment where we can coordinate these changes that will source the docs from this branch: https://h2wt5zfin2hzahss2jnqonnka.js.wpenginepowered.com/

**Comment by RossoMaguire** [comment]
Hey @JEverhart383 thanks for that! I have added the `config.json` now. Let me know if there is anything else you need

**Review by alvarocavalcanti — DISMISSED** [review]
LGTM
High Noon

File: atlas-commerce-connector/atlas-commerce-blocks.mdx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+---
+slug: /atlas-commerce-connector/atlas-commerce-blocks
+title: Atlas Commerce Connector for BigCommerce - Atlas Commerce Blocks
+description: Learn how to build content with BigCommerce products in the WordPress editor.
+---
+
+**Note**: These tutorials assume that you have already [configured the plugin with your BigCommerce API Account credentials](connecting-bigcommerce.mdx) and [performed a product sync](performing-sync.mdx).
+
+Atlas Commerce Blocks lets you place product content synced from BigCommerce in your WordPress content using the WordPress editor data from BigCommerce into WordPress. Using the familiar blocks editorr, you can build pages and posts with product content.
```
Comment (mikepawlinski) [inline atlas-commerce-connector/atlas-commerce-blocks.mdx:9]: 1. You know, I'm not native speaker, this first sentence sounds really hard to me. Any chance we could break it apart? 

Something like
> Atlas Commerce Blocks is a plugin that <...>. It is possible thanks to <...>. 

2. `editorr` typo

File: atlas-commerce-connector/atlas-commerce-blocks.mdx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+## Install the Atlas Commerce Connector plugin
+
+Atlas Commerce Blocks comes pre-installed with the Atlas Commerce Blueprint option via our User Portal and via WP Local.
+
+![Blueprint Plugins](../images/blueprint-plugins.png)
+
+- Check out the demo of [Atlas Commerce Blueprint](#)
+- Try out a [sandbox version of Atlas](#) where you can install Atlas Commerce Blueprint
```
Comment (mikepawlinski) [inline atlas-commerce-connector/atlas-commerce-blocks.mdx:18]: the links are coming later right?

File: atlas-commerce-connector/atlas-commerce-blocks.mdx (hunk 3 above)
Comment (RossoMaguire) [reply atlas-commerce-connector/atlas-commerce-blocks.mdx:9]: Ahh nice one! Cheers Mike

**Review by alvarocavalcanti — DISMISSED** [review]
LGTM

**Comment by RossoMaguire** [comment]
Hey @JEverhart383 we are good to go on this now if you could take another look. Cheers

**Review by alvarocavalcanti — APPROVED** [review]
LGTM

**Review by mikepawlinski — APPROVED** [review]

**Review by JEverhart383 — APPROVED** [review]
This is a great start to docs for this product. Great work y'all, excited to see what the community feedback is like!

**Pull request merged**